crates/miden-lib/src/account/auth/mod.rs

```diff
@@ -58,24 +58,27 @@ impl From<RpoFalcon512> for AccountComponent {
 ///
 /// The storage layout is:
 /// - Slot 0(value): Public key (same as RpoFalcon512)
-/// - Slot 1(value): Number of trigger procedures
+/// - Slot 1(value): [0, 0, allow_unauthorized_output_notes, num_tracked_procs]
```

Comment:
Oh nice, thanks for making this explicitly into a `WORD`, I realize now "Number of trigger procedures" wasn't really hinting at this being a full `WORD` (even if it's just one field element in practice) 👍🏼